### PR TITLE
TC09-User can change password

### PR DESCRIPTION
**Pull request by sattdn210106**

### CODE REVIEW

File: src/main/java/page_objects/ChangePasswordPage.java [hunk 1]
```diff
@@ -1,4 +1,53 @@
...
+    private WebElement getBtnChangePassword() {
+        return BrowserHelper.getDriver().findElement(btnChangePassword);
+    }
+
+    private WebElement getLblSuccessfulMsg() {
+        return BrowserHelper.getDriver().findElement(lblSuccessfulMsg);
+    }
+
+    //Methods
+
```
Comment (tanmle) [inline src/main/java/page_objects/ChangePasswordPage.java]: redundant line

File: src/test/java/tests/ChangePasswordTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        homePage.gotoChangePasswordPage();
+
+        String newPassword = "123456789";
+        changePasswordPage.changePassword(password, newPassword, newPassword);
+
+        String expectedSuccessfulMsg = "Your password has been updated!";
+        String actualSuccessfulMsg = changePasswordPage.getSuccessfulMsg();
+
+        Assert.assertEquals(actualSuccessfulMsg, expectedSuccessfulMsg, "Successful message is incorrect");
```
Comment (tanmle) [inline src/test/java/tests/ChangePasswordTest.java:37]: you should check that after changing password, user is able to login with new password as well.

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/main/java/page_objects/ChangePasswordPage.java (hunk 1 above)
Comment (sattdn210106) [reply src/main/java/page_objects/ChangePasswordPage.java]: Thanks for your review! I fixed this mistake. Please review it

File: src/test/java/tests/ChangePasswordTest.java (hunk 2 above)
Comment (sattdn210106) [reply src/test/java/tests/ChangePasswordTest.java:37]: Thanks for your review! I fixed this mistake. Please review it

**Review by tanmle — APPROVED** [review]

**Pull request merged**